[**LithiumX v1.0.2**](../README.md)

***

[LithiumX](../globals.md) / Exception

# Interface: Exception

Defined in: [src/Structures/Utils.ts:305](https://github.com/anantix-network/LithiumX/blob/791eed01fbe9f7030525ce976bc687f47cb06e89/src/Structures/Utils.ts#L305)

## Properties

### cause

> **cause**: `string`

Defined in: [src/Structures/Utils.ts:308](https://github.com/anantix-network/LithiumX/blob/791eed01fbe9f7030525ce976bc687f47cb06e89/src/Structures/Utils.ts#L308)

***

### message

> **message**: `string`

Defined in: [src/Structures/Utils.ts:306](https://github.com/anantix-network/LithiumX/blob/791eed01fbe9f7030525ce976bc687f47cb06e89/src/Structures/Utils.ts#L306)

***

### severity

> **severity**: [`Severity`](../type-aliases/Severity.md)

Defined in: [src/Structures/Utils.ts:307](https://github.com/anantix-network/LithiumX/blob/791eed01fbe9f7030525ce976bc687f47cb06e89/src/Structures/Utils.ts#L307)
